package.json

```diff
@@ -4,13 +4,13 @@
   "private": true,
   "scripts": {
     "start": "node node_modules/react-native/local-cli/cli.js start",
-    "test": "jest"
+    "test": "jest --forceExit"
   },
   "dependencies": {
     "firebase": "3.6.9",
     "moment": "^2.17.1",
     "native-base": "2.0.7",
-    "react": "15.4.0",
+    "react": "^15.4.2",
     "react-native": "0.41.2",
```

Comment:
I think it's safe to update react version and jest. The issues seem to stem from react-native versions, native-base, and react-native-maps.